vmware/vcd-ext-sdk
# [VACE-2733] [@vcd/schematics] Automate Building Process

Include @vcd/plugin-builders installation in the plugin seed
to reduce the stepps needed to bring the things up and running.

Testing Done:
- Verify when the user name comntains spaces they are converted to "-"
- Verify plugin builders are installed on plugin seed and not if the package
is already installed, i.e it's in the package json.
- Verify if the version of the builders is different then this specified in
the schematis options, the version specifeid, will be installed and the old
one will be overrided.
- Verify the standalone plugin builders package works with the new
ng add schema
- Verify on each run individual plugin builder config is generated

Signed-off-by: Nikola Vladimirov Iliev <nvladimirovi@vmware.com>
